silk/models.py

```diff
@@ -231,6 +231,7 @@ def bulk_create(self, *args, **kwargs):
 
 
 class SQLQuery(models.Model):
+    id = models.AutoField(primary_key=True)
```

Comment:
instead of this 2  changes the app module with appconfig and default key would be appropriate.

Reply:
in that case this PR would be same as #495 ? Was it reverted only because it was "self-merged", I assumed it was because issue was with new "apps.py" file.

Reply:
@Archmonger What additional components in this PR (except for README change, for which another PR could have been filed) would have been needed in it?

As per the review done, if we do those changes then this PR would be identical to #495.

I only wrote this PR because @auvipy told "yeah gonna revert this PR. we need different approch for handling this" he also reviewed with comment "bigAutoField will be used for new django projects autogen primary key, that should not effect this package, for the change we had we should generate new migrations which will need to be supplied with this package.". Therefore, I concluded that the approach we are taking is to have have explicit ids for two models (rest already use explicit ids).

After writing this PR, it is reviewed that need to go back to the #495, therefore I suggest you "revert the revert of #495". Or if we want to go with this (explicit ids in models) approach we should work on this PR. And about the README correct, I will file new PR, if that is agreed upon.

It would not make sense to write an identical PR. Hope that makes sense.

Reply:
first of all you shouldn't have merged that PR. that needed thorough review. that is not how open source contributions are made. and no one told you to follow this approach. in django core your contributions would went through several round of review before actually get merged.